lib/plaid/user.rb

```diff
@@ -211,8 +211,8 @@ def mfa_step(info = nil, send_method: nil, options: nil)
       end
       conn = Connector.new(product, :step, auth: true)
 
-      # Use PATCH if we are in context of User#update.
-      response = if @mfa_patch
+      # Use PATCH if the {update: true} was passed in the options Hash
+      response = if options && options[:update]
```

Comment:
What is an example of when patch magic work? The @mfa_patch does not persist between the request to update user and the subsequent request to mfa_patch right?